std/json.d

```diff
@@ -230,6 +230,43 @@ struct JSONValue
         return store.floating;
     }
 
+    /***
+     * Value getter/setter for boolean stored in JSON.
+     * Throws: `JSONException` for read access if `this.type` is not
+     * `JSONType.true_` or `JSONType.false_`.
+     */
+    @property inout(bool) boolean() inout pure @safe
```

Comment:
Why inout? This should suffice: `@property bool boolean() const pure @safe`. That ensures no surprise with functions that initialize an `auto` with the result of the call.

Reply:
You're right I changed it to const. I'll make another PR to change the other getters as well.

Reply:
#6683